src/rtp/ext.rs

```diff
@@ -636,8 +728,112 @@ pub struct ExtensionValues {
     pub mid: Option<Mid>,
     #[doc(hidden)]
     pub frame_mark: Option<u32>,
+
+    /// User values for [`ExtensionSerializer`] to parse into and write from.
+    pub user_values: UserExtensionValues,
+}
+
+/// Space for storing user extension values via [`ExtensionSerializer`].
+#[derive(Clone, Default)]
+pub struct UserExtensionValues {
+    map: Option<AnyMap>,
```

Comment:
here, I think option is not needed

Reply:
You mean an empty HashMap isn't allocating anything anyway?

Reply:
yes